dev/TelemetryTest/wil/telemetry/MicrosoftTelemetry.h

```diff
@@ -0,0 +1,238 @@
+﻿#pragma once
+/* ++
+
+Copyright (c) Microsoft Corporation.  All rights reserved.
+
+Module Name:
+
+    MicrosoftTelemetry.h
```

Comment:
This should *definitely not* be committed here. Please remove, and then we'll need to rotate the telemetry identifiers when you bring it back in a different form.